app/src/sharedTest/java/org/oppia/android/app/testing/InputInteractionViewTestActivityTest.kt

```diff
@@ -736,15 +760,13 @@ class InputInteractionViewTestActivityTest {
   }
 
   @Test
-  // TODO(#973): Fix InputInteractionViewTestActivityTest
-  @Ignore
   fun testRatioInputInteractionView_withInputtedText_hasCorrectPendingAnswer() {
     val activityScenario = ActivityScenario.launch(
       InputInteractionViewTestActivity::class.java
     )
     onView(withId(R.id.test_ratio_input_interaction_view))
       .perform(
-        typeText(
+        editTextInputAction.appendText(
```

Comment:
This fix makes the test case take forever when it is run using Espresso.  
@rt4914 any idea?

Reply:
Okay, so I thought there's some problem with my laptop, I was trying running the test on espresso and it didn't stop for an hour